src/main/resources/application.properties

```diff
@@ -159,4 +159,7 @@ nae.field-runner.cache-size.namespace-functions=500
 # Filter export file name
 nae.filter.export.file-name=filters.xml
 
-#server.port=8080
\ No newline at end of file
+#server.port=8080
+
+# Authorities
+nae.authority.authorizing-objects=EXAMPLE_AUTHORITY_1,EXAMPLE_AUTHORITY_2
```

Comment:
should these examples be included in the `application.properties`?

Reply:
It was only test, I removed it.